diff --git a/node_modules/jsonresume-theme-actual/assets/styles.css b/node_modules/jsonresume-theme-actual/assets/styles.css
index 2f79d8e..a5e498b 100644
--- a/node_modules/jsonresume-theme-actual/assets/styles.css
+++ b/node_modules/jsonresume-theme-actual/assets/styles.css
@@ -254,7 +254,7 @@ li {
   column-gap: 10px; }
   .skills .container .article {
     break-inside: avoid-column;
-    width: 120px;
+    min-width: 120px;
     padding-top: 20px; }
     .skills .container .article span {
       display: block;
diff --git a/node_modules/jsonresume-theme-actual/assets/template.pug b/node_modules/jsonresume-theme-actual/assets/template.pug
index 67a8c92..2fff329 100644
--- a/node_modules/jsonresume-theme-actual/assets/template.pug
+++ b/node_modules/jsonresume-theme-actual/assets/template.pug
@@ -1,10 +1,11 @@
 doctype html
 html(lang="en")
     head
+        link(rel="icon", href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22><text y=%22.9em%22 font-size=%2290%22>👨‍💻</text></svg>")
         meta(charset="utf-8")
         style
             include styles.css
-        title Actual theme for JSON Resume
+        title=`${resume.basics.name} – ${resume.basics.label}`
     body
         div(class='content')
             include pug/basics
